src/Xamarin.Android.Build.Tasks/Tasks/GenerateResourceDesigner.cs

```diff
@@ -136,10 +136,17 @@ public override bool RunTask ()
 					ITaskItem item = new TaskItem (assemblyPath);
 					assembly.CopyMetadataTo (item);
 					assemblies.Add (item);
-					string alias = assembly.GetMetadata ("Aliases");
-					if (!string.IsNullOrEmpty (alias)) {
-						foreach (var a in alias.Split (new [] {','}, StringSplitOptions.RemoveEmptyEntries))
-							aliases.Add (a.Trim ());
+					string aliasMetaData = assembly.GetMetadata ("Aliases");
+					if (!string.IsNullOrEmpty (aliasMetaData)) {
+						foreach (var alias in aliasMetaData.Split (new [] {','}, StringSplitOptions.RemoveEmptyEntries)) {
+							string aliasName = alias.Trim ();
+							// dont emit an `extern alias global` as it is implicitly done.
+							if (string.Compare ("global", aliasName, StringComparison.OrdinalIgnoreCase) == 0)
+								continue;
+							// only add the first alias for each reference.
+							aliases.Add (aliasName);
+							break;
```

Comment:
No need to adjust this unless it sounds like a good idea. When I was skimming these lines quickly while rereading the PR, I realized my eye paused on this comment for a moment because `aliases.Add()` by itself could in theory add more than just 1 alias, so a tiny idea popped to mind that the comment might look nice moved one line down:

```
aliases.Add (aliasName);
// only add the first alias for each reference.
break;
```